twentytwentyone-blocks/assets/css/style-shared.css

```diff
@@ -11,13 +11,31 @@ body {
 h1,
 .has-huge-font-size,
 .has-gigantic-font-size {
-	font-weight: var(--wp--custom--font-weight-light);
+	font-weight: var(--wp--preset--font-weight-light);
```

Comment:
Can this live in theme.json too?

Reply:
https://github.com/WordPress/theme-experiments/issues/97 I made an issue for the headings a made a note to do that

Reply:
Thanks. Should we remove it from this PR then?

Reply:
Issue for the headings in GB: https://github.com/WordPress/gutenberg/issues/22641 

I'm removing all of the heading rules then.